src/layout/Header/Navbar/index.jsx

```diff
@@ -222,12 +224,32 @@ const list = [
 ];
 
 const Navbar = () => {
-  const fullName = 'Tuan';
-  const total = 10;
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const { token, userInfo } = useSelector((state) => state.userReducer);
+  const { cart } = useSelector((state) => state.cartReducer);
 
-  const handleLogout = () => {};
-  const renderCart = () => {};
+  const handleLogout = () => {
+    dispatch(logoutAction());
```

Comment:
```suggestion
    dispatch(logout());
```
Xem lại cách đặt tên nhé, vì những thứ như này là không cần thiết ấy, vì action này rõ ràng nó nằm ở bên trong user action rồi.

Nó giống như kiểu, thay vì
```
const user = {
   userName: ''.
   userAge: '',
   userAddress: ''
}
```
thì chỉ cần
```
const user = {
   name: ''.
   age: '',
   address: ''
}
```